README.md

```diff
@@ -1,2 +1,115 @@
-# KokkosP Profiling Tools
-Performance profiling tools that can be optionally integrated into Kokkos-based applications and libraries.
+
+
+# Kokkos Tools
+
+Kokkos Tools provide a light weight set of profiling and debug utilities, which interface with instrumentation hooks built directly into the Kokkos runtime. Compared to 3rd party tools these tools can provide much cleaner, context specific information: in particular, they allow kernel centric analysis and they use labels provided to Kokkos objects (kernel launches and views).
+
+Under most circumstances the profiling hooks are compiled into Kokkos executables by default. That means the tools work for your existing Kokkos applications, assuming that the profiling hooks version is compatible with the tools version. No recompilation or changes to your build procedures are required.
+
+Note: `Kokkos_ENABLE_LIBDL` must be on to load profiling hooks dynamically. It should be on by default however.
```

Comment:
When spelled this way, it refers to the Kokkos configuration option.  Is that the intent?

If one really want to, one can:
* either check whether it was indeed enabled by looking whether `KOKKOS_ENABLE_LIBDL` is defined in `KokkosCore_config.h`
* or, better, from Kokkos 3.7 onwards, print the configuration (by passing the `--kokkos-print-configuration` command-line argument or setting the `KOKKOS_PRINT_CONFIGURATION=1` environment variable) and look for `KOKKOS_ENABLE_LIBDL`.

Reply:
I believe it is an option, but I don't know that we want it to be. I will be checking this. 

I think that the above documentation (two bullets you have) is what we want to put in the README whatever the case.